src/ifcblenderexport/blenderbim/bim/export_ifc.py

```diff
@@ -587,21 +587,6 @@ def get_door_attributes(self):
     def get_window_attributes(self):
         return self.get_predefined_attributes("window")
 
-    def get_predefined_attributes(self, attr):
-        results = {}
-        for filename in Path(self.data_dir + attr + "/").glob("**/*.csv"):
-            with open(filename, "r") as f:
-                type_name = filename.parts[-2]
-                pset_name = filename.stem
-                results.setdefault(type_name, []).append(
-                    {
-                        "ifc": None,
-                        "raw": {x[0]: x[1] for x in list(csv.reader(f))},
-                        "pset_name": pset_name.split(".")[0],
-                    }
-                )
-        return results
-
```

Comment:
This function is unrelated to IFCCSV. It is part of the IFC Export, please undo all changes to this file.